src/NServiceBus.Core/Features/FeatureActivator.cs

```diff
@@ -36,7 +36,7 @@ public void Add(Feature feature)
             }));
         }
 
-        public FeaturesReport SetupFeatures(IConfigureComponents container, PipelineSettings pipelineSettings)
+        public FeaturesReport SetupFeatures(IConfigureComponents container, PipelineSettings pipelineSettings, RoutingComponent routing)
```

Comment:
I don't like the special treatment of the routing component. I don't think there is enough unique stuff in it to justify a different treatment than e.g. transport.